src/containers/MapExplorer-SelectorPanel.jsx

```diff
@@ -340,4 +330,7 @@ class SelectorPanel extends Component {
 SelectorPanel.propTypes = {
   dispatch: PropTypes.func.isRequired
 };
-export default connect()(SelectorPanel);
+
+//Don't subscribe to the Redux Store, but gain access to dispatch() and give
+//this component's parent access to this component via getWrappedInstance()
+export default connect(null, null, null, { withRef: true })(SelectorPanel);
```

Comment:
Useful note on this: the `{withRef:true}` option lets an external component (e.g. the parent) access the current component, even after the component has been wrapped up by a `connect`.